### ISSUE TITLE
Lens causes local bash history to be truncated to 500 lines

### ISSUE THREAD

**Issue by istepaniuk** [kontena/lens#7872]
**Describe the bug**
When opening a terminal, Lens executes bash with an `--init-file` parameter, which overrides the local and global .bashrc files. 

This causes bash to use built-in defaults for $HISTSIZE and related variables (which is 500 lines). Consequently, bash truncates the .bash_history file to those 500 lines.

**To Reproduce**
1. Have your ~/.bashrc export $HISTSIZE to a large value, e.g. 1000.
2. Run several commands in an interactive bash shell so you fill your history file. 
3. Verify that $HISTCMD is ~1000
4. Open Lens and launch a local terminal within Lens, then close it.
5. Launch a new interactive bash shell and verify the problem: $HISTCMD is now 500. Half of your history is gone!

**Expected behavior**
Lens should not override the bash initialization file, the history should be preserved.

**Environment**
- Lens Version: 6.5.2
- OS: Debian 12
- Installation method: .deb package

**Comment by Jeansen** [comment]
I cannot reproduce this. Lens 6.5.2 on Debian (Unstable). Did you build Lens yourself?

**Comment by istepaniuk** [comment]
> Did you build Lens yourself?

Yes. I can also reproduce on [OpenLens 6.5.2.309](https://github.com/MuhammedKalkan/OpenLens).

[This line of code](https://github.com/lensapp/lens/blob/f1a960fd785b62a118acd8b1525d879f39917e21/packages/core/src/main/shell-session/local-shell-session/local-shell-session.ts#L62) adds `--init-file`, I can see Lens runs `/bin/bash --init-file .bash_set_path` and the bash man page states:

> --init-file --rcfile
> Execute commands from file instead of the system wide initialization file /etc/bash.bashrc and the standard personal initialization file ~/.bashrc if the shell is interactive.

Do you perhaps have `HISTSIZE` and `HISTFILESIZE` also set elsewhere, even for non-interactive shells?

**Comment by Jeansen** [comment]
Hm..interesting. I build Lens myself with jenkins within a docker container. Container is: `electronuserland/builder:16`
And then I build it with `npm run build:app`

@istepaniuk If you want, I can provide my build in a google drive for you to test.
